test/integration/bad/function-signatures/overloading/stanc.expected

```diff
@@ -0,0 +1,219 @@
+  $ ../../../../../../install/default/bin/stanc no_minimum_dae.stan
+Semantic error in 'no_minimum_dae.stan', line 38, column 12 to column 59:
+   -------------------------------------------------
+    36:    array[4] vector[3] y_hat;
+    37:    {
+    38:      y_hat = dae(chem_dae, yy0, yp0, t0, ts, theta, x, 0, 0);
+                     ^
+    39:    }
+    40:  
+   -------------------------------------------------
+
+No unique minimum promotion found for function "chem_dae".
+Overloaded functions must not have multiple equally valid promotion paths.
+This function has several:
+(real, vector, vector, array[] real, array[] real, real, int) => vector
+(real, vector, vector, array[] real, array[] real, int, real) => vector
+Consider defining a new signature for the exact types needed or
+re-thinking existing definitions.
+  $ ../../../../../../install/default/bin/stanc no_minimum_promotion.stan
+Semantic error in 'no_minimum_promotion.stan', line 12, column 9 to column 17:
+   -------------------------------------------------
+    10:  
+    11:  model {
+    12:     print(foo(1,2));
+                  ^
+    13:  }
+   -------------------------------------------------
+
+No unique minimum promotion found for function "foo".
+Overloaded functions must not have multiple equally valid promotion paths.
+For args (int, int), this function has several:
```

Comment:
Oh I see you do what I asked about above here, is there a reason this does not happen for the DAE solver?

Reply:
Yeah, if the user directly calls a function it was easy to print what they supplied. For all the variadic functions however, the function being passed is called internally and which arguments it receives is variable. I could probably do some special casing for each kind of variadic function to figure it out, but it was very complicated when I tried (stuff like the optional tolerances don't get passed to the user function, for example)